# This message is a submessage of PhantomIntelligencePixel and
# is not intended to be used separately.

uint16 id
uint8 confidence_level
float64 intensity
float64 acceleration
float64 distance
float64 speed
